<launch>

  <node pkg="actionlib_movement" type="exploration_client" name="exploration_client" output="screen"/>

  <node pkg="actionlib_movement" type="movement_server" name="movement_server" output="screen"/>


</launch>
